src/bin/composer-requirements.py

```diff
@@ -5,7 +5,11 @@
 
 
 def merge_requirements(env_data, local_data):
-    env_require = env_data['require']
+    try:
```

Comment:
You can also do 
```python
if not require in env_data:
    print('...')
    exit(0)
```

Reply:
Yes, indeed. And that looks slightly simpler, but I wanted to keep it consistent with the rest of the module code.

Reply:
Well for me that's actually a reason to not use try catch. Using too many try catches will make a file very unreadable.